src/Nri/Ui/Checkbox/V6.elm

```diff
@@ -369,6 +373,100 @@ checkboxUnchecked =
         |> Icon
 
 
+checkboxCheckedHighlighted : Icon
+checkboxCheckedHighlighted =
```

Comment:
❓ why is the check itself a complete different icon?

Reply:
This was the asset I received from design. We could also style the existing checked icon based on the current theme.

Reply:
Also, I wonder if the only SVG assets we really need in this module are the lock and the checkmark. The rest of the checkbox could be CSS.